### PR TITLE
Adds space to the bottom of 22-1995 intro page

### PR DESCRIPTION
**Pull request by cvalarida**
Matches the space at the bottom of the intro page for form 22-1990.

Wasn't really sure where the css should go, so it's in _shame.scss for now.

Closes department-of-veterans-affairs/vets.gov-team#974

### CODE REVIEW

File: src/sass/_shame.scss [hunk 1]
```diff
@@ -31,6 +31,11 @@ body .row {
   border: 1px solid $color-gray-lightest;
 }
 
+.schemaform-intro {
```
Comment (jbalboni) [inline src/sass/_shame.scss]: I think you can put this in edu-benefits.scss for now. In the near future we'll need to pull all the schemaform specific stuff into it's own file, but it's all in edu-benefits for the moment.

**Review by jbalboni — COMMENTED** [review]
Can you post a screenshot of the intro page with your change? We try to do that for any UI changes.

**Comment by cvalarida** [comment]
Before margin:
1995-before-margin


After margin:
1995-after-margin

**Comment by jbalboni** [comment]
@goldenmeanie Does that look ok to you?

**Comment by goldenmeanie** [comment]
screen shot 2017-02-03 at 5 17 27 pm

**Review by jbalboni — APPROVED** [review]

**Pull request merged**